### PR TITLE
Solutions to R Exercise

### PR DESCRIPTION
**Pull request by lixin-source**
I completed some of the exercises & would like some feedback!

### CODE REVIEW

File: week1/citibike.R [hunk 1]
```diff
@@ -1,48 +1,62 @@
...
+
+# count the number of trips (= rows in the data frame)
+# nrow(trips)
+
+# find the earliest and latest birth years (see help for max and min to deal with NAs)
+# max(trips$`birth year`)
+# min(trips$`birth year`)
+
+# use filter and grepl to find all trips that either start or end on broadway
+# View(filter(trips, grepl("Broadway", `start station name`)))
```
Comment (jaredleekatzman) [inline week1/citibike.R:32]: How would you combine these two statements so that it returns one data frame?

File: week1/citibike.R [hunk 2]
```diff
@@ -1,48 +1,62 @@
...
+# View(filter(trips, grepl("Broadway",`end station name`)))
+
+# do the same, but find all trips that both start and end on broadway
+# View(filter(trips, grepl("Broadway", `start station name`), grepl("Broadway",`end station name`)))
+
+# find all unique station names
+# View(unique(trips[c(`start station name`)]))
+# View(unique(trips[c(`end station name`)]))
```
Comment (jaredleekatzman) [inline week1/citibike.R:40]: Again, how do you return 1 'answer' / 1 list?

Are these two lists the same?

File: week1/citibike.R [hunk 3]
```diff
@@ -1,48 +1,62 @@
...
+
+# count the number of trips by gender, the average trip time by gender, and the standard deviation in trip time by gender
+# do this all at once, by using summarize() with multiple arguments
+# group_by(trips, gender) %>%
+# summarize(num_of_trips = n(), mean_of_trips = mean(tripduration/60), sd(trips$gender))
```
Comment (jaredleekatzman) [inline week1/citibike.R:45]: What is  `sd(trips$gender)` doing? Taking the standard deviation of the genders of the trip?
How would you find the standard deviation of trip time?

**Review by jaredleekatzman — COMMENTED** [review]
Looks like a good start! Left some comments on corrections and improvements

**Comment by jhofman** [comment]
Closing these dangling 2021 PRs out to reset for 2022

**Pull request closed without merging**